File: app/src/main/java/com/gals/prayertimes/ui/Splashscreen.java
package com.gals.prayertimes.ui;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.gals.prayertimes.utils.DataManager;
import com.gals.prayertimes.utils.UtilsManager;


public class Splashscreen extends AppCompatActivity {

    UtilsManager tools;
    Intent       toMain;
    //TODO: change to database.
    //TODO: handle exceptions well.
    //TODO: Testing.
    //TODO: remove old code.
    //TODO: background services.


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(com.gals.prayertimes.R.layout.activity_splashscreen);

        toMain = new Intent(this, MainActivity.class);
        tools = new UtilsManager(getBaseContext());

        // Change Statusbar color
        tools.changeStatusBarColor(getWindow());

        new DataManager(toMain,
                        this.getBaseContext(), false).execute("", "", "");

        /*if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            prayer.setPrayersAlarms();
            Log.i("ALARM", "From Splash Screen");
        }*/

    }
}

File: app/src/main/java/com/gals/prayertimes/ui/Splashscreen.kt
package com.gals.prayertimes.ui

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.gals.prayertimes.R
import com.gals.prayertimes.utils.DataManager
import com.gals.prayertimes.utils.UtilsManager

class Splashscreen : AppCompatActivity() {
    lateinit var tools: UtilsManager
    lateinit var toMain: Intent

    //TODO: change to database.
    //TODO: handle exceptions well.
    //TODO: Testing.
    //TODO: remove old code.
    //TODO: background services.
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splashscreen)
        toMain = Intent(
            this,
            MainActivity::class.java
        )
        tools = UtilsManager(baseContext)

        /**Change Status bar color*/
        tools.changeStatusBarColor(window)
        DataManager(
            toMain,
            this.baseContext,
            false
        ).execute(
            "",
            "",
            ""
        )
    }
}